services/ui-src/src/utils/testing/setupJest.tsx

```diff
@@ -42,18 +42,59 @@ export const mockLDFlags = {
   set: mockFlags,
 };
 
-// USERS
+// AUTH
+
+jest.mock("aws-amplify", () => ({
+  Auth: {
+    currentSession: jest.fn().mockReturnValue({
+      getIdToken: () => ({
+        getJwtToken: () => "eyJLongToken",
+      }),
+    }),
+    currentAuthenticatedUser: () => {},
+    configure: () => {},
+    signOut: async () => {},
+    federatedSignIn: () => {},
+  },
+  API: {
+    get: () => {},
+    post: () => {},
+    put: () => {},
+    del: () => {},
+    configure: () => {},
+  },
+  Hub: {
+    listen: jest.fn(),
+  },
+}));
+
+// ROUTER
+
+export const RouterWrappedComponent: React.FC = ({ children }) => (
+  <Router>{children}</Router>
+);
+
+// LAUNCHDARKLY
+
+export const mockLDClient = {
+  variation: jest.fn(() => true),
+};
 
-export const mockNoUser: UserContextShape = {
-  user: undefined,
+// FORM
+export * from "./mockForm";
+
+// USER STATES
+
+export const mockNoUserStore: MfpUserState = {
+  user: null,
   showLocalLogins: true,
-  logout: async () => {},
+  setUser: () => {},
+  setShowLocalLogins: () => {},
   loginWithIDM: () => {},
-  updateTimeout: async () => {},
```

Comment:
@braxex @BearHanded my biggest concern / question is about removing these from Context going forward and how they'll be used in the store.